modules/resetdigitalBidAdapter.js

```diff
@@ -0,0 +1,144 @@
+
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'resetdigital';
+
+function getDomain () {
+  if (!utils.inIframe()) {
+    return window.location.hostname
+  }
+  let origins = window.document.location.ancestorOrigins
+  if (origins && origins.length > 0) {
+    return origins[origins.length - 1]
+  }
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [ 'banner', 'video' ],
+  isBidRequestValid: function(bid) {
+    return (bid.params.pubId !== null || bid.params.zoneId !== null);
```

Comment:
This logic is specifically looking for null values, but not defining either of these params would result in undefined which passes this check and allows the ad units bid request. Can this be updated to check for all empty values?